js/apps/admin-ui/src/clients/utils.ts

```diff
@@ -4,7 +4,8 @@ import type { TFunction } from "i18next";
 /**
  * Checks if a client is intended to be used for authenticating a to a realm.
  */
-export const isRealmClient = (client: ClientRepresentation) => !client.protocol;
+export const isRealmClient = (client: ClientRepresentation) =>
```

Comment:
Let's add an explicit signature to this function.
```suggestion
export const isRealmClient = (client: ClientRepresentation): boolean =>
```

Reply:
Done, thanks!